samples/client/petstore/csharp-dotnet2/OpenAPIClientTest/Lib/OpenAPIClient/README.md

```diff
@@ -53,12 +53,12 @@ namespace Example
             Configuration.Default.AccessToken = "YOUR_ACCESS_TOKEN";
 
             var apiInstance = new PetApi();
-            var pet = new Pet(); // Pet | Pet object that needs to be added to the store
+            var body = new Pet(); // Pet | Pet object that needs to be added to the store
```

Comment:
I'm don't know about this `body` (and many more below). @wing328: I'm guessing they are your additional renames squashed into the cherry-picked typo corrections

Reply:
Good catch. Those changes are not related to your commit. It's due to an enhancement we made to retain the body parameter name defined in OpenAPI v2 spec.